eZ/Bundle/EzPublishCoreBundle/Resources/views/default/content/full.html.twig

```diff
@@ -0,0 +1,5 @@
+<h2>{{ ez_content_name(content) }}</h2>
+{% for field in content.fieldsByLanguage %}
+    <h3>{{ field.fieldDefIdentifier }}</h3>
+    {{ ez_render_field(content, field.fieldDefIdentifier) }}
+{% endfor %}
```

Comment:
> What about also displaying children ( line ), at least if "isContainer" ?

Well, these are default templates only. Purpose of them is mainly to avoid having an exception blowing up if you don't have defined templates for given content.
But recent refactoring in this PR make it possible for one to define custom default templates with custom controllers... However, I really don't see the point to do this by default.